frontend/src/config.json

```diff
@@ -1,22 +1,20 @@
 {
   "metadata" : {
     "data_server"   : "https://covid-19-data-india-storage.mybluemix.net",
-    "link_to_data"  : null,
-    "link_to_slack" : null,
-    "link_to_code"  : null
+    "link_to_data"  : "https://ibm.box.com/s/3d5qzl3e6mdnh9eel7l8nub3p6hfsjrd",
```

Comment:
this should link to the following url: https://www.dropbox.com/s/hbe04q6vtzapdam/covid-india.db?dl=1